### DISCUSSION TITLE
Is there a way to speed up the version update in f-droid store?

### DISCUSSION THREAD

**Discussion by Kutabu** [ochadenas/cpudefense#127] [Q&A] [answered]
I am not familiar with f-droid as a developer, but I have noticed that it sometimes takes up to a week before an update can be downloaded from the platform. Is there a way to speed up the process so that the new build appears faster in the f-droid repo?

I could of course also get the version from GitHub, but because of the different signatures, the local data is not transferred to the new version. Parallel installation is not possible on my Android.

**Comment by ochadenas** [comment, marked as answer]
That's a question I have asked myself more than once. In my experience it is _at least_ a week. May as well be two.

I think that as developer, you don't have any influence on the build process. They have lots of apps to build and verify and their server resources are limited, so it takes a lot of time. And sometimes there are bugs in the build process that stop the whole process.

And besides, currently the app does not build on fdroid (the pipeline throws an error), and I don't know why :( I hope to get that sorted out soon.

**Comment by Kutabu** [comment]
Okay, thanks! I read a little bit in the f-droid wiki. They say, that one week is the usual time for an update and it seems, that there is no (easy) way to speed things up.
_Source: https://gitlab.com/fdroid/wiki/-/wikis/FAQ_

**Comment by ochadenas** [comment]
At least, the app is building now. Whatever caused the problem ... The whole build process is quite complicated in my opinion, and if something goes wrong, you have little chances to correct it or at least start over from the beginning. It just doesn't work, and nobody knows why.

I'm not quite sure whether there will be a 1.31, or whether this version is skipped and the 1.32 will directly appear on fdroid. I've pushed the 1.32 yesterday evening, so we'll just have to wait.

**Comment by ochadenas** [comment]
I was surprised that 1.33 *and* 1.32 did appear on fdroid. Let's see how it works next time ...
